<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsClass</class>
 <widget class="QWidget" name="SettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ark Pixel 12px monospaced zh_cn</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings üéÆüïπÔ∏è‚öôÔ∏è</string>
   </property>
  </widget>
  <widget class="QLabel" name="showlabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Â∞èÈ∏ü</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BirdcomboBox">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ËÉåÊôØ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BackgroundcomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Âú∞Èù¢</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="GroundcomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,10">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ÁÆ°ÈÅì</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="PipecomboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,10">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Èü≥‰πê</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="BGMcomboBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>230</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Èü≥Êïà</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SoundOnradioButton">
        <property name="text">
         <string>ÂºÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="SoundOff">
        <property name="text">
         <string>ÂÖ≥</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Èü≥‰πê</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BGMOnradioButton">
        <property name="text">
         <string>ÂºÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="BGMOffradioButton">
        <property name="text">
         <string>ÂÖ≥</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>239</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="ConfirmpushButton">
      <property name="text">
       <string>Á°ÆËÆ§</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ResetpushButton">
      <property name="text">
       <string>ÈáçÁΩÆ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RecoverpushButton">
      <property name="text">
       <string>ÊÅ¢Â§çÈªòËÆ§</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>158</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="ClassicpushButton">
      <property name="text">
       <string>Classic</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="AngryBirdpushButton">
      <property name="text">
       <string>Angry Bird</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="IronManpushButton">
      <property name="text">
       <string>Iron Man</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="MinecraftpushButton">
      <property name="text">
       <string>Minecraft</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
